auth/checkToken.js

```diff
@@ -0,0 +1,19 @@
+const token = require( 'jsonwebtoken' )
+require( 'dotenv' ).config()
```

Comment:
Yep, we'll need it to help us with our authentication endpoint, which will be in the next PR.